README.md

```diff
@@ -129,6 +129,31 @@ Spin up a real browser that gives you an interface to run individual E2E feature
 npm run test:e2e:ui
 ```
 
+#### End-to-End tests (via Cypress) using dev-upstream AP Tools
+This enables us to locally simulate the e2e test run in the Circle-CI pipeline
+
+First start the ap-tools using
+```
+ap-tools server stop --clear-databases
+ap-tools server start --local-ui --local-api-dev-upstream
+```
+
+* The end-to-end tests run against our local API which then integrates with the dev upstream services and the dev database due to this param `-local-api-dev-upstream`
+* Once AP is running successfully, we can run one of the two npm scripts below which both use the `e2e.dev.env` config file (instead of `e2e.local.env` config file)
```

Comment:
can we please rename the config file e2e.dev-upstream.env for consistency with cas1?